onnxscript/values.py

```diff
@@ -207,7 +207,12 @@ def adapt(input: ExtendedModeValue) -> EagerModeValue:
         elif input is None:
             return None
         elif isinstance(input, list):
-            return [adapt(elt) for elt in input]
+            # orginal code:
+            # return [adapt(elt) for elt in input]
```

Comment:
IIUC, the purpose of this function is to make sure each element is a Tensor. If a user tends to keep the original input data type, this function is not supposed to be called then.

Once this function is call, then it has to make sure each element is a Tensor, not a value.

Reply:
Agree. Can you clarify what is the use-case that is triggering this? ONNX inputs are supposed to: tensors, or lists of tensors, etc. ONNX doesn't support separate scalar types, scalar numbers are encoded as tensors of rank 0. What is the input list [2, 3] supposed to be in the ONNX encoding?